lib/metrics.js

```diff
@@ -15,67 +28,43 @@ module.exports = {
 function prepareData(res) {
   const audits = res.audits;
 
-  const resFMP = audits['first-meaningful-paint'];
-  const resFMPext = resFMP.extendedInfo;
-  const resSI = audits['speed-index-metric'];
-  const resSIext = resSI.extendedInfo;
-  const resTTI = audits['time-to-interactive'];
-  const resTTIext = resTTI.extendedInfo;
-
   const colorP0 = 'yellow';
   const colorP2 = 'green';
   const colorVisual = 'blue';
 
-  const timestamps = [{
-    title: 'Navigation Start',
-    name: 'navstart',
-    value: resFMPext && resFMPext.value.timestamps.navStart
-  }];
+  const timings = [];
+  metricsDefinitions.forEach(metric => {
+    try {
+      const resolvedMetric = {
+        title: metric.name,
+        name: metric.id,
+        value: metric.getTs(audits)
```

Comment:
I'm a dumb thing, forgot about minus `navStart` 😞 
Lets keep it here.
Thanks, it should work perfectly 👍